README.md

```diff
@@ -7,9 +7,59 @@
 
 ## Installation
 
-This package is **in development** and will **break often**.  You can install it from the REPL like so:
+This package is **in development** and will **break often**. In order to start using `GeoMakie` make sure to also add a backend (GLMakie, CairoMakie, WGLMakie), namely:
+
+```julia
+] add GLMakie GeoMakie
+```
+
+You may check the installed versions with:
+
+```julia
+] st GLMakie GeoMakie
+```
+
+Start using the package:
+
 ```julia
-]add GeoMakie
+using GLMakie, GeoMakie
 ```
 
-Please see the documentation for examples and basic usage.
\ No newline at end of file
+# Quick start
+
+The following examples are supposed to be self-explanatory. For further information [check out the documentation here!](https://juliaplots.org/GeoMakie.jl/stable/)
+
+### A simple map
+
+```julia
+fig = lines(GeoMakie.coastlines(); color = :black, axis = (; type = GeoAxis))
+save("coastlines.png", fig; resolution = (600,400))
+fig
+```
+
+The GeoAxis default's projection is [Equal Earth](https://proj.org/operations/projections/eqearth.html).
+
+<img src="./assets/coastlines.png">
+
+### A more colorful map with a different projection can be done with:
```

Comment:
Why not add this to the runnable docs instead.,..?

Reply:
this is not meant to be a large collection of use cases, 3 at the most. That's why we have the docs for. The idea is to have an easier entry point for new users. Copy/paste 3 examples and see how things work without much external knowledge and doing the extra click. Extra clicks always decreases the interest.

Reply:
All of this is fine and I for sure agree with. But what you said is not an argument for putting it on the readme instead of the docs, which is what I am arguing for. What is your argument for putting these on the readme specifically? I don't think the argument "a new user cannot click on the Documentation blue button" is convincing. The downsides of: 

1. Having to maintain both docs and readme
2. Remembering to update readme (I promise you, this will not happen in the future)
3. Spreading information over two sources: docs and readme

are for me good arguments to put these in the docs instead.

Reply:
-  argument for putting it?
Easy/quick entry point to the library, that's all.

And, actually, no updating the readme for this examples is the goal or as less as possible. Package converge/stability. Of course I'm getting the idea from https://github.com/gcalderone/Gnuplot.jl#quick-start, where those examples had been hardly updated in the last 1~2 years and still work as they are. In that particular case people from other fields don't need to know at all specifically the Julia syntax, all that needs to be known is in there(easy entry point).

Reply:
IMO there should be enough on any readme for a quick start.  Once you have a plot up you can look at the docs for the rest.  But the specific methods of how to do things can definitely go to the docs - we should just mention those in the readme.